a10_octavia/controller/worker/tasks/utils.py

```diff
@@ -60,3 +64,11 @@ def meta(lbaas_obj, key, default):
     except Exception:
         return default
     return meta_json.get(key, default)
+
+
+def get_a10_network_driver():
```

Comment:
I think these changes are already in master now

Reply:
Done.